### PR TITLE
Added API/OT status aside for Trust Tokens article

### PR DESCRIPTION
**Pull request by samdutton**

### CODE REVIEW

**Comment by samdutton** [comment]
Sorry @heyawhite — somehow wound up with two branches for this update :/.

samdutton-patch-4 and samdutton-trust-tokens-status.

File: site/en/docs/privacy-sandbox/trust-tokens/index.md [hunk 1]
```diff
@@ -6,16 +6,21 @@ subhead: >
...
+API status updates will be provided here and on [Chrome Platform Status](https://www.chromestatus.com/feature/5078049450098688).
+{% endAside %}
+
 ## Implementation status
 
-* [In origin trial](https://web.dev/origin-trials/) Chrome 84 to 101.
-* [Register for the trial](/origintrials/#/view_trial/2479231594867458049).
-* [Demo](https://trust-token-demo.glitch.me/).
+* [In origin trial](https://web.dev/origin-trials/) Chrome 84 to 101: [now closed](/origintrials/#/view_trial/2479231594867458049).
+* [Demo](https://trust-token-demo.glitch.me/): this is not working, since the origin trial is now closed, but you can still view the code.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/trust-tokens/index.md:23]:
```suggestion
* [Demo](https://trust-token-demo.glitch.me/): this no longer works, as the origin trial is closed. You can still view the code.
```

**Review by heyawhite — APPROVED** [review]
Small simplification suggestion on one line.

@anusmitaray FYI for Trust Tokens content.

File: site/en/docs/privacy-sandbox/trust-tokens/index.md [hunk 2]
```diff
@@ -114,8 +116,8 @@ For more detail about the JavaScript calls in this example, see [Sample API usag
 
 ## Engage and share feedback
 
-* **Origin trial**: Register and take part in the [Chrome origin trial](/origintrials/#/view_trial/2479231594867458049).
-* **Demo**: Try out trust token [issuance and redemption](https://trust-token-demo.glitch.me/).
+* **Origin trial**: Now [closed](/origintrials/#/view_trial/2479231594867458049).
+* **Demo**: The Trust Tokens origin trial has closed, so the [demo](https://trust-token-demo.glitch.me/) won't work, but you can still view code for trust token issuance and redemption.
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/trust-tokens/index.md:120]:
```suggestion
* **Demo**: The Trust Tokens origin trial has closed, so the [demo](https://trust-token-demo.glitch.me/) won't work, but you can still view the code.
```

**Review by heyawhite — APPROVED** [review]

File: site/en/docs/privacy-sandbox/trust-tokens/index.md [hunk 3]
```diff
@@ -6,22 +6,24 @@ subhead: >
...
-{% Aside %}
-This article previously referred to Private State Tokens as "Trust Tokens". The Trust Token API has been renamed to the Private State Token API to better showcase its privacy and utility benefits.
+This article previously referred to Private State Tokens as "Trust Tokens". The Trust Token API has been renamed to the
+Private State Token API to better showcase its privacy and utility benefits.
 
 Check out [Trust Tokens renamed Private State Tokens](/blog/rename-trust-tokens) for more information.
+
+API status updates will be provided here and on [Chrome Platform Status](https://www.chromestatus.com/feature/5078049450098688).
+
 {% endAside %}
```
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/trust-tokens/index.md:21]:
```suggestion
{% Aside %}
This article previously referred to Private State Tokens as "Trust Tokens". The Trust Token
API has been renamed to the Private State Token API to better showcase its privacy and
utility benefits.

Check out [Trust Tokens renamed Private State Tokens](/blog/rename-trust-tokens) for more information.
{% endAside %}
```

**Pull request merged**